requirements.in

```diff
@@ -1,13 +1,15 @@
 click
 docker-py
-PyYAML
-requests
+#PyYAML
+#requests
 retrying
 six
 tqdm
 
 scrapinghub>=2.0.3
 
+pip<19.3
```

Comment:
It seems that in the recent pip, the `main` function was moved from [pip._internal.__init__](https://github.com/pypa/pip/blob/19.2.3/src/pip/_internal/__init__.py#L53-L77) to [pip._internal.main](https://github.com/pypa/pip/blob/19.3/src/pip/_internal/main.py#L22-L47). It's not the best idea to rely on such internal functions, but I'd say there's a simpler way to fix this without restrictions on pip version
```
try:
    from pip import main as pip_main
except:
    try:
        from pip._internal.main import main as pip_main
    except:
        from pip._internal import main as pip_main
```

Reply:
Yeah this is good observation! I should have looked into this. Though I'm not sure for now how to write tests for this. :sweat_smile: